src/Tests/dotnet-test.Tests/GivenDotnetTestContainsMSBuildParameters.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+using FluentAssertions;
+using Microsoft.DotNet.Cli.Utils;
+using System.IO;
+using System;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Cli.Test.Tests
+{
+    public class GivenDotnetTestContainsMSBuildParameters : SdkTest
+    {
+        private const string TestAppName = "VSTestMSBuildParameters";
+        private const string MSBuildParameter = "/p:Version=1.2.3";
+
+        public GivenDotnetTestContainsMSBuildParameters(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [InlineData($"{TestAppName}.csproj")]
+        [InlineData(null)]
```

Comment:
We hit different branches in the `dotnet test` command logic based on if there is a project/sln/dll specified or not, so we need to have test cases for both.

Reply:
Shall we have a 3rd entry for the `.dll` case?

Reply:
The .dll case doesn't hit the same code paths at all, and doesn't forward MSBuild parameters (because there's no build happening) so at best it would be a negative test. Is that valuable?

Reply:
Doesn't feel valuable.